perfzero/lib/perfzero/utils.py

```diff
@@ -445,12 +445,12 @@ def print_thread_stacktrace():
     traceback.print_stack(frame)
 
 
-def instantiate_benchmark_class(benchmark_class, output_dir, root_data_dir):
+def instantiate_benchmark_class(benchmark_class, output_dir, root_data_dir, tpu):
```

Comment:
I think this will break the call from perfzero/lib/benchmark.py - mind checking other usages I might've missed?

https://github.com/tensorflow/benchmarks/search?q=instantiate_benchmark_class&unscoped_q=instantiate_benchmark_class

Reply:
Thanks for catching this.

I updated perfzero/lib/benchmark.py. I also searched for other usage instances and did not find any.